src/index.html

```diff
@@ -21,7 +21,9 @@
 <body>
 
     <div id="content"></div>
-
+    <!-- this file will throw an error because it doesn't exist yet, but it is necessary to let him empty, 
+    cordova will generate it in the future. If you don't need the cordova's features just remove it -->
+    <script type="text/javascript" src="cordova.js"></script>
```

Comment:
Maybe we could left this as a comment. I think the default is the web. What do you think?

Reply:
Oh yes, you are right ! And just replace the comment section to something like ```<!-- uncomment for cordova -->``` and update the readme in the cordova section
I can do this quickly if you want

Reply:
yes please, I think that should be ok